### PR TITLE
feat: display shimmer while loading event types

### PR DESCRIPTION
**Pull request by addiegupta**
Fixes: #1573

Changes:
- Add shimmer while loading event types
- Removed unnecessary text attribute
- Refactored a few files and id's

Screenshots for the change:

1573shimmer

### CODE REVIEW

File: app/src/main/res/layout/fragment_search_type.xml [hunk 1]
```diff
@@ -20,6 +20,40 @@
...
+            <include layout="@layout/placeholder_item_event_type" />
+
+            <include layout="@layout/placeholder_item_event_type" />
+
+        </LinearLayout>
+
+
+    </com.facebook.shimmer.ShimmerFrameLayout>
+
+
```
Comment (nikit19) [inline app/src/main/res/layout/fragment_search_type.xml]: extra

File: app/src/main/res/layout/fragment_search_type.xml [hunk 2]
```diff
@@ -20,6 +20,40 @@
...
+            <include layout="@layout/placeholder_item_event_type" />
+
+            <include layout="@layout/placeholder_item_event_type" />
+
+            <include layout="@layout/placeholder_item_event_type" />
+
+        </LinearLayout>
+
+
```
Comment (nikit19) [inline app/src/main/res/layout/fragment_search_type.xml]: extra

File: app/src/main/java/org/fossasia/openevent/general/search/SearchTypeFragment.kt [hunk 3]
```diff
@@ -33,9 +35,20 @@ class SearchTypeFragment : Fragment() {
...
+            .nonNull()
+            .observe(viewLifecycleOwner, Observer { shouldShowShimmer ->
+                if (shouldShowShimmer) {
+                    rootView.shimmerSearchEventTypes.startShimmer()
+                    adapter.clear()
+                } else {
+                    rootView.shimmerSearchEventTypes.stopShimmer()
+                }
+                rootView.shimmerSearchEventTypes.isVisible = shouldShowShimmer
+            })
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/search/SearchTypeFragment.kt]: add blank line

**Comment by aggarwalpulkit596** [comment]
@addiegupta please resolve conflicts

**Comment by aggarwalpulkit596** [comment]
@nikit19 can i take up on this as no activity for the past 4 days

**Comment by addiegupta** [comment]
> @nikit19 can i take up on this as no activity for the past 4 days

Working on it

**Comment by addiegupta** [comment]
@nikit19 changes made

**Comment by aggarwalpulkit596** [comment]
@iamareebjamal should I take up on this resolve the conflicts?

**Comment by aggarwalpulkit596** [comment]
@nikit19 should I fork this and resolve the conflicts ?

**Comment by nikit19** [comment]
closing this as #1687 has been merged

**Pull request closed without merging**